app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -0,0 +1,83 @@
+.gem-c-pagination {
+  display: block;
+  margin-top: $gutter;
+  margin-bottom: $gutter;
+  margin-left: -$gutter-half;
+  margin-right: -$gutter-half;
+}
+
+// scss-lint:disable SelectorFormat
+.gem-c-pagination__list {
+  margin: 0;
+  padding: 0;
+
+  &:after {
+    content: "";
+    display: block;
+    clear: both;
+  }
+}
+
+.gem-c-pagination__item {
+  @include core-16($line-height: (20 / 16));
+  float: left;
+  list-style: none;
+  text-align: right;
+  margin: 0;
+  padding: 0;
+  width: 50%;
+}
+
+.gem-c-pagination__link {
+  display: block;
+  padding: $gutter-half;
+  text-decoration: none;
+
+  &:visited {
+    color: $link-colour;
+  }
+
+  &:hover,
+  &:active {
+    background-color: $canvas-colour;
+  }
+}
+
+.gem-c-pagination__link-title {
+  @include core-27($line-height: (33.75 / 27));
+  display: block;
+}
+
+.gem-c-pagination__item--previous {
+  @include media-down(mobile) {
```

Comment:
I've not seen this media query on GOV.UK before. What's it doing?

Reply:
Thanks Tom, I wondered if it was something like that. I'll have a crack at rewriting these if @tijmenb is happy.

Reply:
Done in https://github.com/alphagov/govuk_publishing_components/pull/332/commits/7149545984b16e8b505ac77729c9e7911bf15f50.

## Before

![before-mob](https://user-images.githubusercontent.com/233676/40657880-48879eb2-6341-11e8-99ce-fa4304bbcb52.png)
![before-wide](https://user-images.githubusercontent.com/233676/40657881-48a131e2-6341-11e8-95ba-d1eaaa5eebfb.png)

## After

![after-mob](https://user-images.githubusercontent.com/233676/40657892-4f54703a-6341-11e8-8bff-67e209f4bcc7.png)
![after-wide](https://user-images.githubusercontent.com/233676/40657893-4f76e44e-6341-11e8-9b0c-c0cc25e880dc.png)